tests/assets/worker/js/module-worker.js

```diff
@@ -3,6 +3,7 @@
 onmessage = function (e) {
   if (e.data.type === 'startAgent') {
     self.NREUM = e.data.payload
+    // Module workers do not support importScripts, need to import directly
     import('../../../../../../../../web-worker-agent').then(() => {
```

Comment:
I know this is unrelated to this PR, but this import path is goofy.

Reply:
I know, it was so confusing and I changed it...which broke the tests. It's actually an import path relative to the script url when loaded in the browser.